deliveries/dauth_network_milestone_1_2.md

```diff
@@ -0,0 +1,41 @@
+# Milestone Delivery :mailbox:
+
+> ⚡ Only the GitHub account that submitted the application is allowed to submit milestones. 
+> 
+> Don't remove any of the mandatory parts presented in bold letters or as headlines! Lines starting with `>`, such as this one, can be removed.
+
+**The delivery is according to the official [milestone delivery guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/Support%20Docs/milestone-deliverables-guidelines.md).**  
+
+* **Application Document:** https://github.com/w3f/Grants-Program/blob/master/applications/dauth_network.md
+* **Milestone Number:** 1 and 2
+
+
+**Deliverables**
+> Please provide a list of all deliverables of the milestone extracted from the initial application and a link to the deliverable itself. Ideally all links inside the below table should include a commit hash, which will be used for testing. If you don't provide a commit hash, we will work off the default branch of your repository. Thus, if you plan on continuing work after delivery, we suggest you create a separate branch for either the delivery or your continuing work. 
+> 
+> If there is anything particular about any of the deliverables we or a future reader should know, use the respective `Notes` column.
+
+**Milestone 1**
+
+| Number | Deliverable | Link | Notes |
```

Comment:
Please change the milestone tables to match the ones declared in the application. See [here](https://github.com/w3f/Grants-Program/blob/5cc3d7198c45d0ea24c8694ff4047024a4f18ed7/applications/dauth_network.md?plain=1#L134C1-L145C1)